### PR TITLE
Architecture Part V

### PR DESCRIPTION
**Pull request by markus2330**
## Basics

- [x] The PR is rebased with current master.
- [x] Details of what you changed are in commit messages.
- [x] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildserver is happy.

## Checklist

- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I fixed all affected decisions
- [ ] I added unit tests for my code
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I translated all strings visible to the user
- [ ] I mentioned [every code or binary](/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)
- [ ] I created left-over issues for things that are still to be done
- [ ] Code is conforming to [our Architecture](/doc/architecture)
- [ ] Code is conforming to [our Guidelines](/doc/guidelines)
      (exceptions are documented)
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)

## Review

- [ ] I've tested the code
- [ ] I've read through the whole code
- [ ] Examples are well chosen and understandable

### CODE REVIEW

File: doc/decisions/uuid.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+## Assumptions
+
+1. we are actually hit by the problem
+2. we cannot easily change the UUIDs later in production
+
+## Solutions
+
+Use UUIDv7
+
```
Comment (Bushuo) [inline doc/decisions/uuid.md:18]: I recently stumbled upon ULIDs, which aim to solve the above problem.
https://github.com/ulid/spec

Comment (markus2330) [reply doc/decisions/uuid.md]: Great, thanks for mentioning. I added it in 1331d7cd10d7599a194a7cf027754eba4d5eefbb

Do you have some experience with it? Would you favor it over UUIDv7?

Comment (Bushuo) [reply doc/decisions/uuid.md]: I think they are basically the same^^ ULIDs are 10 characters shorter. I think going with UUIDv7 should be fine.

**Comment by markus2330** [comment]
jenkins build please

**Review by kitzbergerg — APPROVED** [review]

**Pull request merged**